css/style.scss

```diff
@@ -105,7 +105,7 @@ footer {
 }
 
 .content {
-  max-width: 100%;
+	max-width: 100%;

```

Comment:
Hi @AverageMarcus, this looks to be indented incorrectly. Could we have 2-space soft-tabs for this and `.post_title` please? (https://github.com/jsoxford/jsoxford.github.com/blob/develop/CONTRIBUTING.md#code-style)